ksql-serde/src/test/java/io/confluent/ksql/serde/json/JsonSerdeUtilsTest.java

```diff
@@ -135,4 +147,88 @@ public void shouldNotConvertIncorrectStringToDouble() {
   public void shouldFailWhenConvertingIncompatibleDouble() {
     JsonSerdeUtils.toDouble(true);
   }
+
+  @Test
+  public void shouldNotIncludeValueInExceptionWhenFailingToBoolean() {
```

Comment:
might make sense to make a quick @Parameterized method out of this (we should update JUnit so we get that, otherwise you need that annoying inner class thing)

Reply:
Funny you should say that, but I did just that originally, then reverted as I didn't want the junit update in this PR...